collector/zfs_freebsd.go

```diff
@@ -251,7 +274,9 @@ func (c *zfsCollector) Update(ch chan<- prometheus.Metric) error {
 	for _, m := range c.sysctls {
 		v, err := m.Value()
 		if err != nil {
-			return fmt.Errorf("couldn't get sysctl: %w", err)
+			// warn logging
+			level.Warn(c.logger).Log("name", m.name, "couldn't get sysctl:", err)
```

Comment:
Wondering if we should even log Warn if we expect some sysctls to be missing. @SuperQ wdyt?

Reply:
This log  Warn maybe provided for user prompts

Reply:
We typically use `Debug` to avoid error messages per scrape.

Reply:
Yeah lets change this to Debug

Reply:
Thanks for your code review. 
Done. Change this to Debug log